### ISSUE TITLE
Peer Review - MacOS

### ISSUE THREAD

**Issue by AustinSeto** [OpenLiberty/draft-guide-operator-openshift#5]
Functionality
---
- [x] Check that the guide has working instructions and sample code
- [x] If any URL visits, try them on different browsers: Firefox, Chrome, Safari
  - [x] Firefox
  - [x] Safari
- [x] If any URL visits, try `curl` command where applicable (`curl` not used)

Formatting & Presentation
---

`README.adoc` checks:
- [x] Ensure that command substitutions use the `$(cmd)` format and not ``` `cmd` ``` (N/A)
- [ ] Ensure that the console output of the integrated tests do not include the `[INFO]` tab
- [x] Check the quality and presentation of guide according to [Structure and Style Guideline](https://github.com/OpenLiberty/draft-guides-template/wiki/Guidelines-for-Structure-and-Styles)
- [x] Check that the pom.xml, server.xml, etc files are clean
    - [x] Remove any dependencies and features that are not required
    - [x] Ensure that the parent pom.xml files do not contain any dependencies, and all dependencies are specified in the children pom.xml files
- [x] Examples with the right outcomes are provided
- [x] Example with the wrong outcomes, if any, are provided (N/A)

`pom.xml` checks (if files are present):
- [x] Check that the file follows the template found here: https://github.com/OpenLiberty/draft-guides-template/blob/master/finish/pom.xml
- [x] Check that the following dependencies are included in all of the `pom.xml` files:
```
<!-- Provided dependencies -->
<dependency>
  <groupId>jakarta.platform</groupId>
  <artifactId>jakarta.jakartaee-api</artifactId>
  <version>8.0.0</version>
  <scope>provided</scope>
</dependency>

<dependency>
  <groupId>org.eclipse.microprofile</groupId
  <artifactId>microprofile</artifactId>
  <version>3.3</version>
  <type>pom</type>
  <scope>provided</scope>
</dependency>
```
- [x]  Check that the versions of plugins are directly provided and not fed in by a file property.
  - Not formatted like so:
```
    <properties>
        <!-- Plugins -->
        <version.liberty-maven-plugin>3.2</version.liberty-maven-plugin>
    </properties>

    <!-- Liberty plugin -->
    <plugin>
        <groupId>io.openliberty.tools</groupId>
        <artifactId>liberty-maven-plugin</artifactId>
        <version>${version.liberty-maven-plugin}</version>
    </plugin>
```
  - Should be formatted like so:
```
    <!-- Liberty plugin -->
    <plugin>
        <groupId>io.openliberty.tools</groupId>
        <artifactId>liberty-maven-plugin</artifactId>
        <version>3.2</version>
    </plugin>
```
- [x] Check that there are 4 spaces per indent for proper formatting
- [x] Ensure that the guide is using the latest version of the `liberty-maven-plugin` or `liberty-gradle-plugin` where applicable

Overall checks:
- [x] Check the consistency of guide with the template and other guides
- [x] Check the quality of code according to the best coding practices
- [x] Check that all licensing statements are properly stated in all files, with the correct year (Should be present in all Java files + the `index.html`)
- [x] Check that the directories are properly structured
- [x] Check that some of these `page-tags` are used in a guide: `MicroProfile, Maven, Docker, Kubernetes, Gradle, Java EE, Security, Cloud`. Only these tags are visible on the website. Latest list [here](https://github.com/OpenLiberty/openliberty.io/blob/master/src/main/content/_includes/visible-tags.liquid). 
- [x] Check the `attribution` statement is accurate for the guide
- [x] Verify the `travisTest.sh` script, if any, is accurate and consistent with other guides (N/A, no script)

---

Guide’s contributor’s (if available, otherwise peer tester’s) responsibility: 
- [ ] Additional tests where applicable: 
    - [ ] Define test coverage and review with team (including guide contributor, if available)
    - [ ] Define detail test cases 
    - [ ] Consider corner cases targeting the specific guide
    - [ ] Consider corner cases UI tests
    - [x] Consider testing URL on all browsers, ie, FF, Chrome, Safari
    - [x] Consider testing the `curl` command for URL visits (N/A)
- [ ] Consider building with both Maven and Gradle build tools 
- [x] Testing with different IDEs, ie, Atom, Eclipse (Optional: VS.code, IntelliJ, Microclimate) (N/A)
- [x] Run Acrolinx Checker on draft (above 70 score approximately)
- [x] Consider SEO title and description for the guides 
- [ ] Ensure automated test is enable, set up with Travis CI, and able to schedule tests 
- [x] Run `diff -r start/ finish/` and there's no differences
- [ ] Ensure that the automation tests are able to run when PR is created

Peer Tester’s responsibility: 
- [x] Check the appearance of the guide on test site for the following items: 
    - [x] Table of contents 
    - [x] Headings
    - [x] Paragraphs
    - [x] code snippets
    - [x] outputs
    - [x] links
    - [x] hotspots 
- [x] Test the guide end-to-end with working instruction and sample code
- [ ] Perform all the defined test cases

**Comment by AustinSeto** [comment]
Remove `<failOnMissingWebXml>` tags in pom.xml files
- https://github.com/OpenLiberty/draft-guide-operator-openshift/blob/35ef8275340f773b4aa91de51cbe844d7788835b/start/system/pom.xml#L105
- https://github.com/OpenLiberty/draft-guide-operator-openshift/blob/35ef8275340f773b4aa91de51cbe844d7788835b/start/inventory/pom.xml#L105
- https://github.com/OpenLiberty/draft-guide-operator-openshift/blob/35ef8275340f773b4aa91de51cbe844d7788835b/finish/system/pom.xml#L105
- https://github.com/OpenLiberty/draft-guide-operator-openshift/blob/35ef8275340f773b4aa91de51cbe844d7788835b/finish/inventory/pom.xml#L105

**Comment by AustinSeto** [comment]
Make it more clear that the `git clone` command is also run within the OpenShift console. Currently the note stating commands are run within the OpenShift console is after the section where the clone is instructed.

**Comment by DannyS03** [comment]
# Functionality
## Installing the Open Liberty Operator
- [ ] Instructions on **Using the CLI** and **Using the web console** should be at the beginning of this section and go before the command to create a new project. 

## Deploying the system and inventory applications to OpenShift
### Building and pushing the images
- [ ] add `include::{common-includes}/ol-kernel-docker-pull.adoc[]`

# Formatting & Presentation
## What you’ll learn

- [ ] Perhaps: Using a cloud **orchestrator** helps you to easily scale and manage your microservices in a high-availability setup.
- [ ] Perhaps a sentence before about the benefit of deploying your microservices to the cloud. 

> Using a cloud helps you to easily scale and manage your microservices in a high-availability setup. Sounds a little awkward. 

- [ ] The Open Liberty Operator **provides** a method of deploying and managing Open Liberty applications on OKD or OpenShift clusters. 

- [ ] Using Custom Resource Definitions (CRDs) **,** the Open Liberty Operator introduces a new object, the **`OpenLibertyApplication`**. 

- [ ] The Operator acts as a custom controller **by** continuously comparing the current state and the desired state of the Kubernetes node and deployed OpenLibertyApplications, and reconciling when necessary. 

- [ ] I think this sentence... 
> The Operator acts as a custom controller by continuously comparing the current state and the desired state of the Kubernetes node and deployed OpenLibertyApplications, and reconciling when necessary. 

... should go after  
> The OpenLibertyApplication is a high-level object, which can include regular well-known ‘children’ resources such as Pods, Deployments, Services, ReplicaSets, etc. 
  
- [ ] Perhaps highlighting ‘children’ to `children`, or `child` as the reference to the resource type is something that would be known.
- [ ] The listing of resources should be limited to three if using `etc`. Perhaps: such as Pods, Deployments, Services, etc. 
> The OpenLibertyApplication is a high-level object, which can include regular well-known ‘children’ resources such as Pods, Deployments, Services, ReplicaSets, etc. 

- [ ] Same idea for `parent` and `children` here. 
> Since the Open Liberty Operator controls the ‘parent’ OpenLibertyApplication resource, the Operator also subsequently controls all ‘children’ resources. 


- [ ] You will deploy two microservices, system and inventory **,** to an OpenShift cluster using the Open Liberty Operator. The system and inventory applications will communicate with a Kafka container that you will deploy directly from DockerHub. To learn more about the reactive Java application used in this guide and how it works, check out the Creating reactive Java microservices guide.


## Installing the Open Liberty Operator
- [ ] Log **in to** your OpenShift cluster **;** this guide assumes that all subsequent commands will be run from within the cluster.

- [ ] Instructions on **Using the CLI** and **Using the web console** should be at the beginning of the section and go before the command to create a new project. 

## Deploying the Kafka service to OpenShift

- [ ] If explaining the parameters of the command that deploys ZooKeeper and Kafka, it would help with consistency if the flags for the command are also briefly mentioned. 

<img width="572" alt="image" src="https://user-images.githubusercontent.com/25497083/82425437-822ae100-9a54-11ea-9a48-dd71ead58b73.png">

---

- [ ] You **will** see the following output:

## Deploying the system and inventory applications to OpenShift
- [ ] To deploy the Open Liberty system and inventory applications, you need to package the applications, build the Docker images, push them to the internal OpenShift registry, **and** then deploy the images using the Open Liberty Operator.

### Packaging the applications
- [ ] You can configure OpenShift to build your images from **various** methods.

- [ ] **Navigate to the start directory. Run** the following commands to package the system and inventory applications:

### Building and pushing the images
- [ ] **Next, run** the following commands to view the newly created image streams and build configurations for each application:

- [ ] In this case, the build **inputs** are your `binary` (local) files, which will be streamed to OpenShift for the build.

- [ ] You **will** see the following resources:

- [ ] You **will** see the following output:

### Deploying the images
- [ ] **Next, run** the following command to view your newly created OpenLibertyApplication resources:

## Accessing the inventory application

- [ ] You **will** see an output similar to:

- [ ] Visit the inventory application by **pointing** the following URL **. Ensure** that you substitute the appropriate HOST:

## Scaling up the system application
- [ ] Scaling applications up is easy with **Operators. Simply** update the deploy.yaml file with the replica: n parameter.

- [ ] Revisit the inventory application and you **will** now see three elements:

**Comment by MaiHameed** [comment]
> Tests missing 

Addressing with the `travis-tests` branch

> Update page release date 

This date is unknown, will be updated closer to release date

> In the section "Accessing the inventory application" I was unable to find resources in the guide namespace when getting the routes. 

Did you ensure that you waited until the build(s) for the system/inventory were complete? I added instructions to clarify this.

> Change indentation to 4 spaces in following files:

2 spaces is usually the standard for XML files

> Instructions on Using the CLI and Using the web console should be at the beginning of this section and go before the command to create a new project.

The operator needs to be installed to watch a project namespace, so the project needs to  be created before you install the operator.

> Using a cloud helps you to easily scale and manage your microservices in a high-availability setup.

This was the sentence used in the OpenShift guide + was approved by all reviewers. I'm adding `orchestrator` and leaving as is

> Perhaps highlighting ‘children’ to children, or child as the reference to the resource type is something that would be known.

`Parent` and `Children` aren't real resources, so i enclosed them in quotes to show that I used the word as an analogy for what they represent.

> Log in to your OpenShift cluster ; this guide assumes that all subsequent commands will be run from within the cluster.

This sentence/section has been completely rewritten so this change was made redundant

> If explaining the parameters of the command that deploys ZooKeeper and Kafka, it would help with consistency if the flags for the command are also briefly mentioned.

Not sure what you mean here, do you mean that I should add an explanation for what the `-l` flag does? I already explained the `-e` flag for the Zookeeper app, the same applies for Kafka and won't be necessary to re-explain.

> You will see the following output:

Proper term is `You see` according to ID review.

> Visit the inventory application by pointing the following URL . Ensure that you substitute the appropriate HOST:

The term `visit the URL` is often used across guides, I used it here for consistency.

**Comment by MaiHameed** [comment]
Addressed with #10
